forest/main.py

```diff
@@ -127,7 +127,8 @@ def main(argv=None):
         else:
             # Use dataset interface
             settings = {
-                "pattern": group.pattern
+                "pattern": group.pattern,
+                "color_mapper": color_mapper
```

Comment:
`"label": group.label` could go here

Reply:
In the future there will be muliple color_mappers and dataset specific settings but for now this is the desired approach. Pass in all the settings needed to configure all `Datasets` it's a step in the right direction

Reply:
yes, OK sounds good I will make that change. I couldn't remember what the group label did so I thought I could let it quietly slip into the night :)

Reply:
It's meta-data for the tooltip. I toyed with the idea of using it as a primary navigation key, but now I think that concept needs to be de-coupled, e.g. assign a uid for each dataset on start up